scripts/zones/Mhaura/DefaultActions.lua

```diff
@@ -1,12 +1,16 @@
-local ID = require("scripts/zones/Mhaura/IDs")
-
 return {
-    ['Haigan']        = { event = 10002 },
-    ['Kotan-Purutan'] = { event = 140 },
-    ['Lacia']         = { event = 10021 },
-    ['Numi_Adaligo']  = { event = 50 },
-    ['Phoochuchu']    = { event = 300 },
-    ['Rycharde']      = { event = 69 },
-    ['Take']          = { event = 59 },
-    ['Tonasav']       = { event = 372 },
+    ['Amalanbraux']      = { event = 700 },
+    ['Galdo']            = { event = 650 },
+    ['Hagain']           = { event = 10002 },
+    ['Itzha_Delavhitta'] = { event = 600 },
```

Comment:
There are two event 600s? Is that right? Is this a pair of NPCs that responds with one event when clicking either?

Reply:
pretty sure old pxi and DSP scripts did that in a bunch of places to substitute for event ids they couldn't get or couldn't get working.

on occasion we've capped an event in retail and then watched the client NOT run the event due to some parameter being wrong or missing (ToAU had a few where we HAD to feed the event zeros for unused params, or they just would not load. And guild npc's require new functionality or would freeze)

Reply:
> There are two event 600s? Is that right? Is this a pair of NPCs that responds with one event when clicking either?

yeah those default events were taken from the existing scipt (that was removed)

Reply:
Sounds like we need a cap to see if these are both 600 with args, or if its a long standing bug

Reply:
> Sounds like we need a cap to see if these are both 600 with args, or if its a long standing bug

got it. let me assign a soul in xiweb to get captures :D

Reply:
@zach2good  daywhite from xiweb confirmed the CS already existing are accurate, they both use CS 600

Reply:
![image](https://user-images.githubusercontent.com/60417494/230818327-377c2b72-8410-42c9-af37-cd7f6e623bbd.png)

Reply:
![image](https://user-images.githubusercontent.com/60417494/230818431-aa7ab81b-0072-4ebd-a252-f2762c689e40.png)

Reply:
![image](https://user-images.githubusercontent.com/60417494/230818611-27cea88f-a0a0-45d4-be2c-7557f894c78c.png)

Reply:
_Teo cough's loudly_

Reply:
thanks for checking @WinterSolstice8  :D  
i checked the recording again and you are right, she does CS 900, sorry for mistake @zach2good 

![image](https://user-images.githubusercontent.com/26943220/230821022-e33da5f4-c425-4829-b24a-ef7e2402de04.png)

@TeoTwawki  want a mint? :smile:

Reply:
> @TeoTwawki want a mint? 😄

\*lozenge

Reply:
> > @TeoTwawki want a mint? smile
> 
> *lozenge

Ricolas are yummy :D

Made change to Itzha. Resolving convo

Reply:
a note for future reference: please let reviewers do the resolving.

Reply:
oh crap i didnt know that, i have been doing it all along lol xD thanks for the note, Will make sure to do so in the future. unresolving it now. thanks @TeoTwawki  :D

Reply:
Don't sweat it. its githubs fault, been 6 months since I raised the issue of literally _anybody_ being able to resolve a convo and they didn't even acknowledge the issue report.